### DISCUSSION TITLE
Is there such a thing as an "unswipe" at the breakpoints?

### DISCUSSION THREAD

**Discussion by zonky2** [nolimits4web/swiper#4357] [Q&A]
see 3. example at https://kenwheeler.github.io/slick/
```
settings: "unslick"
```

I have a picture gallery for the desktop display - only on tablet and mobile should it become a slider.

**Comment by zonky2** [comment]
thx to move...

**Reply by benjaminleeschnell** [reply]
This answer was helpful to me: https://github.com/nolimits4web/swiper/issues/3175

**Comment by zonky2** [comment]
@benjaminleeschnell - o.k. that would be a option https://github.com/nolimits4web/swiper/issues/3175#issuecomment-522067213

but with two lines in the config it would be more elegant than all the code
